docs/README.md

```diff
@@ -5,8 +5,8 @@ The website content it is deployed automatically in the `gh-pages` branch by Git
 After cloning the repo (with submodules), just run `make serve` to test the website locally.
 
 ```
-$> git clone --recurse-submodule https://github.com/rancher-sandbox/cos-toolkit
-$> cd cos-toolkit
+$> git clone --recurse-submodule https://github.com/rancher/elemental-toolkit
+$> cd elemental-toolkit
```

Comment:
since we are here.. care to update it to only `make docs-serve` ? no need to cd into docs anymore